bitnami/minio/templates/_helpers.tpl

```diff
@@ -93,9 +93,9 @@ Get the root password key.
 Return true if a secret object should be created
 */}}
 {{- define "minio.createSecret" -}}
-{{- if .Values.auth.existingSecret -}}
+{{- if or (.Values.auth.existingSecret) (eq .Values.auth.useSecret false) -}}
 {{- else -}}
-    {{- .Values.auth.useSecret -}}
+    {{- true -}}
 {{- end -}}
 {{- end -}}
```

Comment:
I know this was like this prior to your PR, but could you please simplify this as:
```
{{- define "minio.createSecret" -}}
{{- if and (not .Values.auth.existingSecret) .Values.auth.useSecret -}}
    {{- true -}}
{{- end -}}
```

This was a really nice catch! Although the value `.Values.auth.useSecret` is of type boolean, what happened here is what I consider of the weirdest behaviors with helm.

What happens with this helpers is the following:
```
{{- define "my_helper" -}}
{{- if .var }}
{{- true -}} <-- when '{{if include "my_helper" .}}' is called, this is interpreted as 'true'
{{- else -}}
{{- false -}} <-- when '{{if include "my_helper" .}}' is called, this is also interpreted as 'true'
{{- end -}}
```

The actual solution to this is to:
```
{{- define "my_helper" -}}
{{- if .var }}
{{- true -}} <-- when '{{if include "my_helper" .}}' is called, this is interpreted as 'true'
{{- end -}} 
{{- end -}}  <-- empty return would be interpreted as 'false'
```

In this case, using empty `if` and return value based on input value only makes logic more complex, leading to this bug you catched.

Reply:
Ah, thank you for providing a more sleek solution, looks even cleaner!

Yes, that's what I tried to say in my PR description. Sorry if that was written in a confusing way. :)